blockchain/blockchain.go

```diff
@@ -395,7 +395,7 @@ func (bc *blockchain) context(ctx context.Context, tipInfoFlag, candidateFlag bo
 func (bc *blockchain) MintNewBlock(
```

Comment:
from naming perspective, I think it not very trivial that `MintNewBlock` doesn't return a block.